src/Container/SimpleApplicationFactory.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/**
+ * @see       https://github.com/mezzio/mezzio for the canonical source repository
+ * @copyright https://github.com/mezzio/mezzio/blob/master/COPYRIGHT.md
+ * @license   https://github.com/mezzio/mezzio/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Mezzio\Container;
+
+use Laminas\Diactoros\Response;
+use Laminas\Diactoros\ServerRequest;
+use Laminas\Diactoros\ServerRequestFactory;
+use Laminas\HttpHandlerRunner\Emitter\SapiEmitter;
+use Laminas\HttpHandlerRunner\Emitter\SapiStreamEmitter;
+use Laminas\HttpHandlerRunner\RequestHandlerRunner;
+use Laminas\Stratigility\MiddlewarePipe;
+use Mezzio\Application;
+use Mezzio\MiddlewareContainer;
+use Mezzio\MiddlewareFactory;
+use Mezzio\Response\ServerRequestErrorResponseGenerator;
+use Mezzio\Router\RouteCollector;
+use Mezzio\Router\RouterInterface;
+use Psr\Container\ContainerInterface;
+
+class SimpleApplicationFactory
```

Comment:
Maybe s/simple/minimal?

Reply:
We must avoid the word "simple" here because it can be simple for you and me but not for others.